apps/aechannel/src/aesc_session_noise.erl

```diff
@@ -211,10 +211,16 @@ accept_tcp(LSock, Port, R, AcceptTimeout) ->
         {ok, _} = Ok ->
```

Comment:
I see the line above (`gen_tcp:accept`) and the origin of the listening socket `LSock` mean that concurrent `gen_tcp:accept` may be called on the same listening socket. I read that [that is ok in recent OTP](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/doc/src/gen_tcp.xml#L128-L133). So the implementation seem sound.